src/image.rs

```diff
@@ -72,13 +72,7 @@ impl<Img> Map<Img> {
         self.map.get_mut(&id)
     }
 
-    /// Inserts the given widget-image pair into the map.
-    ///
-    /// If the map did not already have an image associated with this widget, `None` is returned.
-    ///
-    /// If the map did already have an image associated with this widget, the old value is removed
-    /// from the map and returned.
-    ///
+    /// Inserts the given image pair into the map.
```

Comment:
Ahh thanks for removing the old comment here!

Would you mind adding that this method also returns an `Id` associated with the added image, and that the user must store this in order to use, modify or remove the inserted image?